src/core/reopt.jl

```diff
@@ -387,9 +387,19 @@ function build_reopt!(m::JuMP.AbstractModel, p::REoptInputs)
         end
     end
 	
+	# Include the electric storage cost constants only if the installed_cost_constant or the replace_cost_constant is not zero
+	for b in p.s.storage.types.elec
```

Comment:
@toddleif Out of curiosity, why did you add this here instead of `constraints/storage_constraints.jl`?

Reply:
Thank you for bringing this up. I moved this code to a function in `constraints/storage_constraints.jl`.